pyproject.toml

```diff
@@ -79,8 +79,16 @@ fastapi-utils = { version = "~0.2.1", optional = true}
 
 # optional dependencies for stack recipes
 
+# Optional dependencies for project templates
+copier = ">=7.0.0"
+jinja2-time = "^0.2.0"
+black = "^22.3.0"
+ruff = "^0.0.238"
+autoflake = "^2.0.1"
```

Comment:
I think we don't need autoflake any more. We just removed it for ourselves [in this PR](https://github.com/zenml-io/zenml/pull/1279).

Reply:
Beautiful ! I updated my templates scripts to reflect this. One package less in my requirements.